xcube/core/resample.py

```diff
@@ -63,6 +64,11 @@ def resample_in_time(cube: xr.Dataset,
 
     if var_names:
         cube = select_vars(cube, var_names)
+        
+    if frequency == 'ALL':
```

Comment:
* Please use lowercase letters. i.e. `all`, to distinguish it from other settings, which are abbreviations and ALL is not an abbreviation.
* Update docstring to reflect new possible value `all`.